src/library.js

```diff
@@ -125,6 +126,7 @@ export {
   formatDate,
   formatISODate,
   futureDateValidator,
+  getAge,
```

Comment:
Need to update these function names as well. Please make sure the library builds successfully ("npm run build").

Reply:
done